src/plotting/plotSettings.component.tsx

```diff
@@ -512,11 +552,33 @@ const PlotSettings = (props: PlotSettingsProps) => {
                     padding: 1,
                   }}
                 >
-                  <Typography noWrap>{plotChannel}</Typography>
-                  <StyledClose
-                    aria-label={`Remove ${plotChannel} axis`}
-                    onClick={() => removePlotChannel(plotChannel)}
-                  />
+                  <Typography noWrap>{plotChannel.name}</Typography>
```

Comment:
This has been a pain. The text does not wrap. Therefore, if you click on a really long channel name, the entirety of the label is rendered and the width of the container element is extended to match, creating a horizontal scroll bar. I've tried to fix this by adding `xs` this and `width: 'inherit'` that but had no luck so any help would be appreciated here for style points.

Reply:
Honestly, I think the easiest solution here is to set a sensible `maxWidth` on the `Typography` based on the drawer width - everything else. Experimentally, `190px` works well

Reply:
Guess I was trying to be too clever! `maxWidth="190"` works very well, thank you :)